### PR TITLE
Create new component for completed decisions so they can see all the …

### PR DESCRIPTION
**Pull request by perkinss**
…text of the decision reason

*Issue #:#1100*

*Description of changes:*
Create a new simplified component to view just the completed NRs
Benefit: snapshot tests can easily catch inadvertent changes to the UI
Drawback: still have to remove completed_NR code from CompName component

** also updated the snapshot tests for some other components.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: client/config/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[*]
```
Comment (perkinss) [inline client/config/.editorconfig]: D'oh ignore this
File should not have made it into this commit

File: client/src/components/application/Examine/CompletedCompName.vue [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+        return this.$store.getters.currentState;
+      },
+      userId() {
+        return this.$store.getters.userId;
+      },
+      is_furnished() {
+        if (this.$store.getters.furnished === "Y") return true;
+        return false;
+      },
+      is_approved_expired() {
```
Comment (perkinss) [inline client/src/components/application/Examine/CompletedCompName.vue]: I am not sure the significance of some of the states of these, I copied it from the CompName so I didn't lose anything, but without understanding the business purpose of them.

File: client/test/jest/ComponentSnapshotTests/application/__snapshots__/NameExamination.spec.js.snap [hunk 3]
```diff
@@ -33,6 +33,8 @@ exports[`NameExamination.vue renders a NameExamination view 1`] = `
           class="col"
         >
           <compnameview-stub />
+
```
Comment (perkinss) [inline client/test/jest/ComponentSnapshotTests/application/__snapshots__/NameExamination.spec.js.snap]: oops didn't mean to add an empty comment

File: client/test/jest/ComponentSnapshotTests/application/examine/__snapshots__/Decision.spec.js.snap [hunk 4]
```diff
@@ -19,7 +19,7 @@ exports[`Decision.vue renders a Decision view 1`] = `
           deselectgrouplabel="Press enter to deselect group"
           deselectlabel=""
           limit="99999"
-          limittext="function(t){return\\"and \\".concat(t,\\" more\\")}"
+          limittext="function (t){return\\"and \\".concat(t,\\" more\\")}"
```
Comment (perkinss) [inline client/test/jest/ComponentSnapshotTests/application/examine/__snapshots__/Decision.spec.js.snap]: :-/ another accidental commit.  Jeez I just assumed that anything under "test" was deliberate.

File: client/test/jest/ComponentSnapshotTests/application/stats/__snapshots__/Stats.spec.js.snap [hunk 5]
```diff
@@ -59,7 +59,7 @@ exports[`Stats.vue renders a Stats component 1`] = `
       >
         <b-pagination
           current-page="1"
-          per-page="100"
+          per-page="50"
```
Comment (perkinss) [inline client/test/jest/ComponentSnapshotTests/application/stats/__snapshots__/Stats.spec.js.snap]: something that got missed earlier

**Comment by perkinss** [comment]
> So I'm considering how much functionality is in CompName and I don't think we should duplicate it all. However what you could do is break the decision text stuff out into two new components - the open version and closed (completed) version. Then you're not duplicating all the buttons and functionality that goes along with CompName (which is really much more functionality than comp names, as it was originally intended).

Yes, one of the things I didn't do in this branch was to remove duplicated functionality from CompName.  But I thought I only kept the things that they currently see in a completed NR, such as reopen, cancel NR, and getNext.

I'm not sure whether they are needed, just that they exist now.

**Comment by katiemcgoff** [comment]
As discussed, we are not moving forward with this change, in favour of the simpler implementation.

**Pull request closed without merging**